src/components/xl-newsletter/xl-newsletter.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { StaticImage } from "gatsby-plugin-image";
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+
+import { Close } from "../icons"
+
+import { classnames } from "../../helpers";
+
+import * as styles from "./xl-newsletter.module.scss";
+
+const XlNewsletter = () => (
+  <div className={styles.container} >
+    <div className={styles.logoContainer}>
+      <Link to="/" className={styles.linkLogoXl}>
+        <StaticImage    
+          src="../../../static/images/newsletter-xl/logo-xl.png"
+          alt="Xmartlabs logo"
+          layout="fixed"
+          width={100}
+          height={100}
+        />
+      </Link>
+    </div>
+    <div className={styles.informationContainer}>
+      <h2 className="text__heading__two__neutral100">Xmartlabs´ Newsletter</h2>
+      <p className={classnames("text__paragraph__two__defaultGray", styles.textNewsletter)}>Subscribe to our newsletter and get updates on AI, Computer Vision as well as mobile and web development.</p>
+      <div className={styles.subscribeContainer}>
+        <input placeholder="Type your email..." className={classnames(styles.inputNewsletter, "text__placeholder__grayThree")} />
+        <button className={classnames(styles.buttonNewsletter, "text__paragraph__small__neutral100")} onClick={() => {}}>SUBSCRIBE</button>
+      </div>
+    </div>
+    <Close className={styles.close} />
+  </div>
+);
+
+export { XlNewsletter };
+
+XlNewsletter.propTypes = {
+  onClick: PropTypes.func,
+};
+
+XlNewsletter.defaultProps = {
+  onClick: () => {},
+}
```

Comment:
Remove this

Reply:
All the props or only the defaultProps?

Reply:
All, because this component doesn't have onclick prop

Reply:
done